docs/Getting-Started/Installation/installation.md

```diff
@@ -0,0 +1,77 @@
+# Installing Pop!_OS
+
+The Pop!_OS installer guides the user through setup of basic system settings.
+
+- Verify system and installation requirements.
+- Choose a locale and keyboard language.
+- Perform system hard drive selection and partitioning (advanced).
+- Create a user account.
+- Enable system disk encryptions.
+
+---
+## Requirements
+
+### System Specifications
+
+| Component | Requirement | Recommended |
+|-----------|-------------|-------------|
+| CPU       | 64-bit x86 architecture |     
+| RAM       | 2 GB        | 4 GB        |
+| Drive Storage | 20 GB   |             | 
+
+### Secure Boot
+
+Systems with Secure Boot enabled must disable this feature before installing Pop!_OS. See [Microsoft's official documentation](https://docs.microsoft.com/en-us/windows-hardware/manufacture/desktop/disabling-secure-boot?view=windows-10) for this procedure.
+
+### Installation Media
+
+The Pop!_OS.iso can be easily flashed to a USB drive. See [Create a Bootable Pop!_OS USB.](/Getting-Started/Create-bootable-media/create-bootable-usb.md)
+
+---
+## Installation
+
+1. Choose a language.
+
+![Choose Language](/images/installation/choose-language.png)
+
+2. Select a locale.
+
+![Language Region](/images/installation/language-region.png)
+
+3. Select a keyboard input language.
+
+![Keyboard Language](/images/installation/keyboard-language.png)
+
+4. Select a keyboard layout.
+
+![Keyboard Layout](/images/installation/keyboard-language2.png)
+
+5. Choose the `Clean Install` option for a standard installation. This is the best option for new Linux users. 
```

Comment:
We should note that is will erase the contents of the target drive. Users should have robust backups from their macOS or Windows install first.